src/js/edu-benefits/1990e/config/form.js

```diff
@@ -31,7 +48,36 @@ const formConfig = {
       pages: {
         applicantInformation: {
           path: 'applicant-information',
-          title: 'Applicant information'
+          title: 'Applicant information',
+          initialData: {},
+          uiSchema: {
+            applicantFullName: fullName.uiSchema,
+            applicantSocialSecurityNumber: ssn.uiSchema,
+            applicantDateOfBirth: currentOrPastDate.uiSchema('Applicant date of birth'),
+            applicantGender: {
+              'ui:widget': 'radio',
+              'ui:title': 'Applicant gender'
+            },
+            relationship: {
+              'ui:widget': 'radio',
+              'ui:title': 'What is your relationship to the service member whose benefit is being transferred to you?'
+            }
+          },
+          schema: {
+            type: 'object',
+            required: ['applicantFullName'],
+            properties: {
+              applicantFullName: fullName.schema,
```

Comment:
Actually, it's not the veteran in this case, but I'll change it to `relativeFullName` to match the backend schema